trunner/test/test_harness.py

```diff
@@ -0,0 +1,240 @@
+#
```

Comment:
I suggest renaming this file to `harness/test_embdtls.py` or `test_harness_embdtls.py`